### ISSUE TITLE
WCAG issue: Small difference between button and link + hover

### ISSUE THREAD

**Issue by HeleneKassandra** [SpareBank1/designsystem#898]
We have some buttons that look exactly the same as the links. 
This causes an issue as it's hard for users to see the difference between a link and a button with this styling.

This class or styling also has a very poor hover state, making it very hard for users to see if they are hovering the right element. 

Component: TertiaryButton

**Comment by hdaljord** [comment]
I agree, TertiaryButton is not a good button. But I'm not sure that it is a problem that the button looks like a link (i.e. the styling), but I agree that the hover state style is just bearly visible and should be improved.

@sirioc and @Heilysto, maybe you could have a look at this?

**Comment by HeleneKassandra** [comment]
> I agree, TertiaryButton is not a good button. But I'm not sure that it is a problem that the button looks like a link (i.e. the styling), but I agree that the hover state style is just bearly visible and should be improved.
> 

The issue with it looking like a link is that it's difficult for users to see if clicking it will navigate them to a new location, or complete/start an action.
For users this can be very important, as navigating to a new page, might mean that they have the re-enter a bunch of information in forms etc.

**Comment by hdaljord** [comment]
That is not a problem just with this button with the usages of all buttons as they all accept other elements, like anchors. There are several examples of using buttons as links in SB1 applications. I.e. buttons have been used as links. If we should do this correctly no buttons should allow for anchor elements. 

I do not agree on this usage, I only note that it has been practised.

**Comment by HeleneKassandra** [comment]
> That is not a problem just with this button with the usages of all buttons as they all accept other elements, like anchors. There are several examples of using buttons as links in SB1 applications. I.e. buttons have been used as links. If we should do this correctly no buttons should allow for anchor elements.
> 
> I do not agree on this usage, I only note that it has been practised.

I get what you're saying, but the issue you mention is code related and not design. The frontend team (me included) are looking into the design system to hopefully solve some of those issues on a component level in the future.

But when it comes to the tertiary button I think it has a problematic design, that goes beyond the freedom we have in the components. The design itself makes it hard for users and developers to differentiate between button and link.
Personally I think that making a clearer distinction between link and button in the design system, will make it easier to make the right choices when designing and creating UI's. 

Either way, it's an interesting discussion so thank you for your input!

**Comment by hdaljord** [comment]
I agree with you :) It would be nice to address all the similar issues with buttons and links and add some text into the design guide about how they should be used at the same time to make both usage and design consistent across applications.

**Comment by ivarni** [comment]
> The issue with it looking like a link is that it's difficult for users to see if clicking it will navigate them to a new location, or complete/start an action.

This is my problem with it as well. Links are for navigation and having something that looks like a link sets an expectation that pressing it will cause a navigation event. It would be better to let links look like links and buttons look like buttons.

**Comment by kwltrs** [comment]
Tertiary button har litt annen styling med focus, padding, inline-block osv. enn knappene, ikke minst etter siste redesign.
